src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectServices/VsCoreProjectSystemReferenceReader.cs

```diff
@@ -207,7 +249,7 @@ private IEnumerable<Reference> GetVSProjectReferences()
             return excludedReferences;
         }
 
-        private bool IsProjectReference(Reference3 reference, Common.ILogger logger)
+        private bool IsProjectReference(Reference reference, Common.ILogger logger)
```

Comment:
Why did you change this one? It's still passing same `Reference3` type pointer here.

Reply:
I think it was mostly the process than the result that led to this changes. 
It was just something when I was refactoring, coming up with cleanest way of writing the code. 

I reverted it.